ddtrace/_trace/tracer.py

```diff
@@ -127,6 +128,17 @@ def __init__(self) -> None:
         # traces
         self._pid = getpid()
 
+        # Enable logging on native components
+        if config._native_logging:
```

Comment:
we should probably move this to:

https://github.com/DataDog/dd-trace-py/blob/c72189816067e03c68c9451dc75ca3a9dc1b08cf/ddtrace/bootstrap/preload.py